tutorials/radiant-mlhub-on-demand-training-data.ipynb

```diff
@@ -0,0 +1,1281 @@
+   "metadata": {},
+   "source": [
+    "As we can see from the metadata for the Xarray above, the Landsat 8 scene has a significantly larger geographic footprint, `~20,000 x ~20,000 pixels`, compared to `120 x 120 pixels` for the Sentinel-2 chips that were prepared for the dataset. We need to crop/mask the image down so they represent the exact same terrain.\n",
+    "\n",
+    "Luckily, the `bounds_latlon` parameter of `stackstac` makes it easy to crop the image to this size automatically for all bands/assets requested."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0045f54d-5076-4a14-9292-3dbdde339cdc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l8_cropped = stack(\n",
+    "    items=ItemCollection([best_l8_match]),\n",
+    "    assets=LANDSAT_8_RGB_BANDS,\n",
+    "    bounds_latlon=source_item.bbox,\n",
+    "    resolution=10,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0f0600e6-3bb8-4da9-90b2-ee4ec45ea227",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l8_cropped"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d7a5e313-0a2f-410e-8339-9630ca87dd78",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l8_cropped[0].plot(col=\"band\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bc46c85b-daa9-44df-9be7-62dfa1234b25",
+   "metadata": {},
+   "source": [
+    "Now we have a cropped Landsat 8 chip that spatially and temporally matches our Sentinel-2 source imagery and label sample from the BigEarthNet dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3a70b063-d273-4aee-864b-07e318388890",
+   "metadata": {},
+   "source": [
+    "### Launch a Dask gateway cluster for parallel processing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cb650d80-bf1a-4aad-8f8b-08a612e28aae",
+   "metadata": {},
+   "source": [
+    "We will use Dask to optimize our data processing of hundreds of Landsat-8 scenes by parallelizing the workflow with a delayed computation graph. The Dask Client schedules, runs the delayed computations, and gathers the results, while the Dask Gateway provides a secure and centralized way of managing the multiple client clusters. This is especially useful for running Dask on Planetary Computer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29531759-6d19-4010-8401-eb947a32c515",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = dd_client(\n",
+    "    # you can configure Dask client parameters here\n",
+    ")\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "927a0496-a6c0-4aa2-9181-708904a5bb4f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# this cell will only work on PC or a machine with gateway cluster configured\n",
+    "# gateway = dask_gateway.Gateway()\n",
```

Comment:
IMO: pick whether or not you want this example to run on a cluster. If it runs without a cluster, then we can just remove this.

Reply:
Removed